flaskbb/forum/views.py

```diff
@@ -189,7 +189,6 @@ def form(self):
 
 
 class NewTopic(MethodView):
-    decorators = [login_required]
```

Comment:
Rather than deleting this, it should be changed to `decorators = [`Has('newtopic')]` (can't remember the actual name of the permission off hand)